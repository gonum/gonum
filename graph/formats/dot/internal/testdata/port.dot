digraph {
	A:ne -> B:sw
	C:foo -> D:bar:se
	E:_ -> F
	G:n
	H:e
	I:s
	J:w
	K:nw
	L:c
}
