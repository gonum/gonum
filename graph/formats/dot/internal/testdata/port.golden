digraph {
	A:ne -> B:sw
	C:foo -> D:bar:se
	E -> F
	G:n
	H:e
	I:s
	J:w
	K:nw
	L:c
}
